js/layout-javascript/news-feed-code.js

```diff
@@ -438,9 +438,8 @@ DynamicList.prototype.attachObservers = function() {
       $(this).siblings('.panel-heading').find('.fa-angle-up').removeClass('fa-angle-up').addClass('fa-angle-down');
     })
     .on('click', '.news-feed-comment-holder', function(event) {
-      _this.$container.find('.news-feed-list-wrapper').hide();
```

Comment:
@tonytlwu please confirm you're good with this change

Reply:
I'm happy with it. It was only added as part of https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/427 in [this commit](https://github.com/Fliplet/fliplet-widget-dynamic-lists/commit/21280d7c802780d18319cef254c819e8c6a9e2b0) hidden amongst all other changes. It didn't seem necessary at the time and should be safe to remove.